fixtures/u2_import_path.rb.sem.uast

```diff
@@ -0,0 +1,50 @@
+{ '@type': "module",
+   '@role': [Identifier, Module, Statement],
+   file: { '@type': "begin",
+      '@role': [Block],
+      '@pos': { '@type': "uast:Positions",
+         start: { '@type': "uast:Position",
+            offset: 0,
+            line: 1,
+            col: 1,
+         },
+         end: { '@type': "uast:Position",
+            offset: 57,
+            line: 2,
+            col: 32,
+         },
+      },
+      body: [
+         { '@type': "uast:QualifiedIdentifier",
+            '@pos': { '@type': "uast:Positions",
+               start: { '@type': "uast:Position",
+                  offset: 0,
+                  line: 1,
+                  col: 1,
+               },
+               end: { '@type': "uast:Position",
+                  offset: 24,
+                  line: 1,
+                  col: 25,
+               },
+            },
+            Names: [require],
```

Comment:
It's loading the wrong name here. Should be a `./u2_class_this`, and it's not an element of Names (Names are subsymbols imported from a package), but a Path (package path)

Reply:
Same reason as above, native AST not typing the send_require node correctly.

Reply:
done